website/cue/reference/components/sinks/splunk_hec.cue

```diff
@@ -0,0 +1,47 @@
+package metadata
+
+components: sinks: _splunk_hec: {
+	configuration: {
+		acknowledgements: {
```

Comment:
To clarify, this deviates from other sink's `acknowledgements` configuration style (`acknowledgements.enabled`) because it's automatic, correct? It turns on when response includes an `ackID`. Do we not want to let users opt-out if they don't want this, for performance reasons, etc?

Reply:
Sinks do not normally have acknowledgement configuration. If the protocol allows, they always wait for and check the response from a request.

Reply:
Right, the `splunk_hec` sinks either acknowledge based on an initial HTTP `200 OK` response status or based on this indexer acknowledgements implementation (if an `ackID` is present). I support making the indexer acknowledgements behavior explicitly disable-able since it can add potentially significant overhead to the source. Instead of `acknowledgements.enabled`, maybe this can look like `acknowledgements.indexer_acknowledgements_enabled`.

A current workaround to opt-out would be for users to configure the sink with a Splunk HEC token that has indexer acknowledgements disabled.